File: web3rpc/rpc-specs/code-samples/javascript/kaia/transaction/getDecodedAnchoringTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const hashOfTransaction =
    "0x026b64e16b86633c0199f78f37a64840d3601d83e5c799f115b63024764524ca";

  provider.kaia
    .getDecodedAnchoringTransactionByHash(
      hashOfTransaction,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/transaction/getDecodedAnchoringTransactionByHash.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

hashOfTransaction = "0x72cde80650c7c6745e4cf4c162e9ce1e5542f4d86112925faa6aa75f6a6142ec"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_decoded_anchoring_transaction_by_hash(hashOfTransaction)

print(kaia_response)
